### PR TITLE
ignite: add page

### LINKED ISSUE #1778: page request: ignite
Requesting [infinitered/ignite](https://github.com/infinitered/ignite) page.

### PR DESCRIPTION
**Pull request by wisn**
Fix #1778

----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines

### CODE REVIEW

File: pages/common/ignite.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# ignite
+
+> An unfair headstart for your React Native apps.
```
Comment (agnivade) [inline pages/common/ignite.md]: We try to refrain from using subjective descriptions in our pages. Please change the language to something more objective and formal.

File: pages/common/ignite.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+- Create a new React Native project:
+
+`ignite new {{ProjectName}}`
```
Comment (agnivade) [inline pages/common/ignite.md]: ProjectName => project_name

**Comment by wisn** [comment]
@agnivade I've fixed it

**Comment by agnivade** [comment]
I have removed the comma after React Native to make the sentence look more in tune with the fact that the tool is about React Native and not that React Native is just one aspect of the tool.

**Comment by sbrl** [comment]
Looks ok to me! Thanks, @wisn 😺

**Pull request merged**